### PR TITLE
Adding new fields to json schema for ot_crispr datasets.

### PR DESCRIPTION
**Pull request by DSuveges**
The schema validates the following crispr d/t evidence strings:

```json
{"targetFromSourceId": "YIF1B", "diseaseFromSourceMappedId": "EFO_0000249", "projectId": "OTAR036", "studyId": "OTAR036_TAU_uptake_1", "studyOverview": "Comparison of functional populations of iPSC neurons based on their ability to take up monomeric/aggregated tau protein", "contrast": "TauNEGTransPOS (only transferrin taken up) vs TauPOSTransPOS  (both proteins taken up)", "crisprScreenLibrary": "Kosuke v1.1 (Behan et al., 2018)", "cellType": "iPSC derived cortical neurons", "cellLineBackground": "KOLF2-C1", "direction": "upper tail", "resourceScore": 0.0063227, "datasourceId": "ot_crispr", "datatypeId": "ot_partner"}
{"targetFromSourceId": "YIF1B", "diseaseFromSourceMappedId": "EFO_0003096", "projectId": "OTAR036", "studyId": "OTAR036_TAU_uptake_1", "studyOverview": "Comparison of functional populations of iPSC neurons based on their ability to take up monomeric/aggregated tau protein", "contrast": "TauNEGTransPOS (only transferrin taken up) vs TauPOSTransPOS  (both proteins taken up)", "crisprScreenLibrary": "Kosuke v1.1 (Behan et al., 2018)", "cellType": "iPSC derived cortical neurons", "cellLineBackground": "KOLF2-C1", "direction": "upper tail", "resourceScore": 0.0063227, "datasourceId": "ot_crispr", "datatypeId": "ot_partner"}
{"targetFromSourceId": "TCP10L2", "diseaseFromSourceMappedId": "EFO_0000249", "projectId": "OTAR036", "studyId": "OTAR036_TAU_uptake_1", "studyOverview": "Comparison of functional populations of iPSC neurons based on their ability to take up monomeric/aggregated tau protein", "contrast": "TauNEGTransPOS (only transferrin taken up) vs TauPOSTransPOS  (both proteins taken up)", "crisprScreenLibrary": "Kosuke v1.1 (Behan et al., 2018)", "cellType": "iPSC derived cortical neurons", "cellLineBackground": "KOLF2-C1", "direction": "upper tail", "resourceScore": 0.0063799, "datasourceId": "ot_crispr", "datatypeId": "ot_partner"}
{"targetFromSourceId": "TCP10L2", "diseaseFromSourceMappedId": "EFO_0003096", "projectId": "OTAR036", "studyId": "OTAR036_TAU_uptake_1", "studyOverview": "Comparison of functional populations of iPSC neurons based on their ability to take up monomeric/aggregated tau protein", "contrast": "TauNEGTransPOS (only transferrin taken up) vs TauPOSTransPOS  (both proteins taken up)", "crisprScreenLibrary": "Kosuke v1.1 (Behan et al., 2018)", "cellType": "iPSC derived cortical neurons", "cellLineBackground": "KOLF2-C1", "direction": "upper tail", "resourceScore": 0.0063799, "datasourceId": "ot_crispr", "datatypeId": "ot_partner"}
{"targetFromSourceId": "ABCC9", "diseaseFromSourceMappedId": "EFO_0000249", "projectId": "OTAR036", "studyId": "OTAR036_TAU_uptake_1", "studyOverview": "Comparison of functional populations of iPSC neurons based on their ability to take up monomeric/aggregated tau protein", "contrast": "TauNEGTransPOS (only transferrin taken up) vs TauPOSTransPOS  (both proteins taken up)", "crisprScreenLibrary": "Kosuke v1.1 (Behan et al., 2018)", "cellType": "iPSC derived cortical neurons", "cellLineBackground": "KOLF2-C1", "direction": "upper tail", "resourceScore": 0.0082308, "datasourceId": "ot_crispr", "datatypeId": "ot_partner"}
{"targetFromSourceId": "ABCC9", "diseaseFromSourceMappedId": "EFO_0003096", "projectId": "OTAR036", "studyId": "OTAR036_TAU_uptake_1", "studyOverview": "Comparison of functional populations of iPSC neurons based on their ability to take up monomeric/aggregated tau protein", "contrast": "TauNEGTransPOS (only transferrin taken up) vs TauPOSTransPOS  (both proteins taken up)", "crisprScreenLibrary": "Kosuke v1.1 (Behan et al., 2018)", "cellType": "iPSC derived cortical neurons", "cellLineBackground": "KOLF2-C1", "direction": "upper tail", "resourceScore": 0.0082308, "datasourceId": "ot_crispr", "datatypeId": "ot_partner"}
{"targetFromSourceId": "ART3", "diseaseFromSourceMappedId": "EFO_0000249", "projectId": "OTAR036", "studyId": "OTAR036_TAU_uptake_1", "studyOverview": "Comparison of functional populations of iPSC neurons based on their ability to take up monomeric/aggregated tau protein", "contrast": "TauNEGTransPOS (only transferrin taken up) vs TauPOSTransPOS  (both proteins taken up)", "crisprScreenLibrary": "Kosuke v1.1 (Behan et al., 2018)", "cellType": "iPSC derived cortical neurons", "cellLineBackground": "KOLF2-C1", "direction": "upper tail", "resourceScore": 0.0027475, "datasourceId": "ot_crispr", "datatypeId": "ot_partner"}
{"targetFromSourceId": "ART3", "diseaseFromSourceMappedId": "EFO_0003096", "projectId": "OTAR036", "studyId": "OTAR036_TAU_uptake_1", "studyOverview": "Comparison of functional populations of iPSC neurons based on their ability to take up monomeric/aggregated tau protein", "contrast": "TauNEGTransPOS (only transferrin taken up) vs TauPOSTransPOS  (both proteins taken up)", "crisprScreenLibrary": "Kosuke v1.1 (Behan et al., 2018)", "cellType": "iPSC derived cortical neurons", "cellLineBackground": "KOLF2-C1", "direction": "upper tail", "resourceScore": 0.0027475, "datasourceId": "ot_crispr", "datatypeId": "ot_partner"}
{"targetFromSourceId": "IL15", "diseaseFromSourceMappedId": "EFO_0000249", "projectId": "OTAR036", "studyId": "OTAR036_TAU_uptake_1", "studyOverview": "Comparison of functional populations of iPSC neurons based on their ability to take up monomeric/aggregated tau protein", "contrast": "TauNEGTransPOS (only transferrin taken up) vs TauPOSTransPOS  (both proteins taken up)", "crisprScreenLibrary": "Kosuke v1.1 (Behan et al., 2018)", "cellType": "iPSC derived cortical neurons", "cellLineBackground": "KOLF2-C1", "direction": "upper tail", "resourceScore": 0.0022693, "datasourceId": "ot_crispr", "datatypeId": "ot_partner"}
{"targetFromSourceId": "IL15", "diseaseFromSourceMappedId": "EFO_0003096", "projectId": "OTAR036", "studyId": "OTAR036_TAU_uptake_1", "studyOverview": "Comparison of functional populations of iPSC neurons based on their ability to take up monomeric/aggregated tau protein", "contrast": "TauNEGTransPOS (only transferrin taken up) vs TauPOSTransPOS  (both proteins taken up)", "crisprScreenLibrary": "Kosuke v1.1 (Behan et al., 2018)", "cellType": "iPSC derived cortical neurons", "cellLineBackground": "KOLF2-C1", "direction": "upper tail", "resourceScore": 0.0022693, "datasourceId": "ot_crispr", "datatypeId": "ot_partner"}
```

### CODE REVIEW

File: opentargets.json [hunk 1]
```diff
@@ -1110,6 +1178,13 @@
         "glioblastoma vs normal"
       ]
     },
+    "crisprScreenLibrary": {
+      "type": "string",
+      "description": "The applied screening library in the CRISPR/CAS9 prject.",
```
Comment (ireneisdoomed) [inline opentargets.json:1183]:
```suggestion
      "description": "The applied screening library in the CRISPR/CAS9 project.",
```

File: opentargets.json [hunk 2]
```diff
@@ -1205,6 +1290,10 @@
         "CHEMBL803"
       ]
     },
+    "geneticBackground": {
+      "type": "string",
+      "description": "Additional genetic background of the applied cell line in the CRISPR screen."
```
Comment (ireneisdoomed) [inline opentargets.json:1295]: Can you provide an example?

**Review by ireneisdoomed — APPROVED** [review]
All good.

File: opentargets.json [hunk 3]
```diff
@@ -592,6 +592,57 @@
...
       "additionalProperties": false
     },
+    {
+      "properties":{
+        "datasourceId": {
+          "const": "ot_crispr"
+        },
+        "cellLineBackground": {
+          "$ref": "#/definitions/cellLineBackground"
+        },
```
Comment (tskir) [inline opentargets.json:602]: (Same nitpick comment about following the alphabetical order)

File: opentargets.json [hunk 4]
```diff
@@ -1181,9 +1254,19 @@
...
         "somatic_mutation",
         "known_drug",
-        "genetic_association"
+        "genetic_association",
+        "ot_partner"
       ]
     },
+    "direction": {
+      "type": "string",
+      "description": "Which end of the distribution the target was picked.",
```
Comment (tskir) [inline opentargets.json:1263]:
```suggestion
      "description": "Which end of the distribution the target was picked from.",
```

File: opentargets.json [hunk 5]
```diff
@@ -1181,9 +1254,19 @@
         "animal_model",
         "somatic_mutation",
         "known_drug",
-        "genetic_association"
+        "genetic_association",
+        "ot_partner"
       ]
     },
+    "direction": {
```
Comment (tskir) [inline opentargets.json:1261]: If this is something specific to CRISPR studies, could we make the field name more specific, for example `crisprDirection`? Or are we expecting to reuse it in other datasources?

**Review by tskir — APPROVED** [review]
Everything looks good save for several minor comments

**Pull request merged**